pages/en/contrib/Governance.md

```diff
@@ -20,18 +20,22 @@ Each repository has one or more lead maintainers responsible for:
 *   Seeking consensus on technical decisions.
 *   Making the final decisions when consensus cannot be achieved.
 
-Maintainers have npm publishing rights for modules and the final say on releasing new versions.
+Maintainers have npm publishing rights for modules and the final say on releasing new versions.  
 
-## Lead maintainers
+Refer to `CODEOWNERS` file in each repository for the list of lead maintainers. 
 
-LoopBack lead maintainers include:
+### Full-time maintainers
 
+Project architects:
 *   [Miroslav Bajtoš](https://github.com/bajtos)
 *   [Raymond Feng](https://github.com/raymondfeng)
-*   [Ritchie Martori](https://github.com/ritch)
-*   [Simon Ho](https://github.com/superkhau)
 
-For more details, see [Functional area owners](functional-area-owners.html).
+Full-time maintainers include:
```

Comment:
I think we should either:
- be explicit about the fact that this list contains people on IBM payroll only
- or include all current maintainers 

I feel the current proposal is making community contributors a kind of second-class maintainers.

It would be great if we could fetch an up-to-date list of people from somewhere else (or point readers to such source), e.g. GitHub's list of people in `@strongloop/loopback-maintainers` teams. Unfortunately, AFAICT, there is no such public view :(

Reply:
thanks for pointing it out.  I didn't realize it will cause this impression!